service/scheduler.go

```diff
@@ -59,18 +59,38 @@ type Scheduler struct {
 	OrchardHost       string
 	OrchardAPIKeyName string
 	OrchardAPIKey     string
+	LockTTL           time.Duration
 }
 
 func (s *Scheduler) Start() {
 	fmt.Println("Scheduler Started")
 	tick := time.Tick(s.Interval)
-	for range tick {
-		fmt.Println("tick")
-		s.scheduleWorkflows(database.GetInstance())
-		s.activateWorkflows(database.GetInstance())
+	lockTtlTick := time.Tick(time.Minute * 5) // TODO: ok to hardcode to 5 mins?
```

Comment:
Hardcoded to check for and delete locks older than LockTTL every 5 minutes. Could also do it every second, or parameterize the interval in the config.

Reply:
Moved the default lockTTL up to 15 mins per our convo